godot-core/src/builtin/plane.rs

```diff
@@ -9,24 +9,26 @@ use std::ops::Neg;
 use godot_ffi as sys;
 use sys::{ffi_methods, GodotFfi};
 
-use super::{is_equal_approx, real, Vector3};
+use super::{is_zero_approx, is_equal_approx, real, Vector3};
+
+pub const CMP_EPSILON: real = 0.00001;
 
 /// 3D plane in [Hessian normal form](https://mathworld.wolfram.com/HessianNormalForm.html).
 ///
 /// The Hessian form defines all points `point` which satisfy the equation
 /// `dot(normal, point) + d == 0`, where `normal` is the normal vector and `d`
 /// the distance from the origin.
 ///
-/// Currently most methods are only available through [`InnerPlane`](super::inner::InnerPlane).
-///
 /// Note: almost all methods on `Plane` require that the `normal` vector have
 /// unit length and will panic if this invariant is violated. This is not separately
 /// annotated for each method.
 #[derive(Copy, Clone, PartialEq, Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[repr(C)]
 pub struct Plane {
+    /// The plane's normal component.
     pub normal: Vector3,
+    /// The plane's d component.
     pub d: real,
```

Comment:
The current documentation has not more information than the field names -- could you instead write what they mean?

Doesn't need to be long, something like "normal vector pointing away from the plane" is enough.